.gitignore

```diff
@@ -2,6 +2,7 @@
 
 # dependencies
```

Comment:
you should include `.ci`

Reply:
`.ci` is there:
https://github.com/src-d/code-annotation/pull/36/files/09f1a6ced524c250859222d93f9916b280c9b5ba#diff-a084b794bc0759e7a6b77810e01874f2R27

```
# ci files
Makefile.main
.ci
bin
```